bitbots_vision/config/visionparams.yaml

```diff
@@ -40,30 +60,19 @@ blue_color_detector_upper_values_h: 116
 blue_color_detector_upper_values_s: 255
 blue_color_detector_upper_values_v: 255
 
-field_boundary_finder_search_method: 'dynamic'  # iteration, reversed, binary or dynamic
-field_boundary_finder_vertical_steps: 30
-field_boundary_finder_horizontal_steps: 30
-field_boundary_finder_roi_height: 1
-field_boundary_finder_roi_width: 1
-field_boundary_finder_roi_increase: 0.08  # increase of kernel radius per pixel
-field_boundary_finder_green_threshold: 150
-field_boundary_finder_precision_pix: 5
-field_boundary_finder_min_precision_pix: 3
-field_boundary_finder_head_joint_threshold: -0.5
+field_boundary_detector_search_method: 'reversed'  # iteration, reversed, binary or dynamic
```

Comment:
why is this changed?

Reply:
Do you mean to `reversed`?
This is a debugging artifact. Could be switched back to `dynamic`.